### PR TITLE
Update start pages for new email signups

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/mk5M2aKD/619-general-design-and-stylistic-updates-to-signup-workflow

Please see the commits for more details. In order to keep the
design changes clear, I've split out refactorings into separate
commits e.g. for code that's been made redundant.

## Before

Screenshot 2020-11-24 at 14 09 55
Screenshot 2020-11-24 at 14 10 09
Screenshot 2020-11-24 at 14 10 25

## After

Screenshot 2020-11-24 at 14 05 08
Screenshot 2020-11-24 at 14 05 30
Screenshot 2020-11-24 at 14 07 48

### CODE REVIEW

File: app/views/content_item_signups/new.html.erb [hunk 1]
```diff
@@ -22,32 +21,27 @@
                    track_category: "email_subscriptions",
                  }) do %>
 
+      <%
+         radio_items = @subscription.child_taxons.each.map do |taxon|
```
Comment (kevindew) [inline app/views/content_item_signups/new.html.erb]: Does this need `each.map` - I'm wondering if it's a mistake or a fruity data structure?

File: app/helpers/taxons_helper.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module TaxonsHelper
```
Comment (kevindew) [inline app/helpers/taxons_helper.rb:1]: Might be that this is hinting towards content item being a model. That seems a later concern though and removing this presenter class looks a nice improvement.

**Review by kevindew — APPROVED** [review]
Looks good to me, potentially a minor thing to fix.

File: app/views/content_item_signups/new.html.erb (hunk 1 above)
Comment (benthorner) [reply app/views/content_item_signups/new.html.erb]: Haha, no - not sure how that got in there. Removed 👍.

File: app/helpers/taxons_helper.rb (hunk 2 above)
Comment (benthorner) [reply app/helpers/taxons_helper.rb:1]: Yeah, I did wonder about that. I think we need some more examples before modelling it, as the current usage is so specific to a particular document type.

File: app/views/content_item_signups/new.html.erb (hunk 1 above)
Comment (kevindew) [reply app/views/content_item_signups/new.html.erb]: It looks like it was in the previous version, so more a c+p issue than you going mad 😁

**Review by kevindew — APPROVED** [review]

File: app/views/content_item_signups/confirm.html.erb [hunk 3]
```diff
@@ -1,5 +1,4 @@
-<% page_heading = "Sign up to get emails" %>
-<% content_for :title, page_heading + ' - ' + @subscription.title %>
+<% content_for :title, t("content_item_signups.confirm.title") %>
 
 <% if live_content_item?(@content_item) %>
   <% content_for :back_link do %>
```
Comment (gavinwye) [inline app/views/content_item_signups/confirm.html.erb:4]: I thought we were removing the back link on all these pages?

File: app/views/content_item_signups/taxon.html.erb [hunk 4]
```diff
@@ -0,0 +1,49 @@
+<% content_for :title, t("content_item_signups.taxon.title") %>
+
+<% if live_content_item?(@content_item) %>
+  <% content_for :back_link do %>
```
Comment (gavinwye) [inline app/views/content_item_signups/taxon.html.erb:4]: Remove?

File: app/views/content_item_signups/confirm.html.erb (hunk 3 above)
Comment (benthorner) [reply app/views/content_item_signups/confirm.html.erb:4]: Indeed we are. That will come in a separate PR, as we made the decision half way through the work.

**Comment by gavinwye** [comment]
✅ looks good to me 👍

**Pull request merged**